docs/DORA.md

```diff
@@ -5,47 +5,49 @@ description: >
 
 ![](https://i.imgur.com/71EUflb.png)
 
-To make DORA even more actionable, there are well-established benchmarks to determine if you are performing at "Elite", "High", "Medium", or "Low" levels. Inside DevLake, you will find the benchmarking table available to assess and compare your own projects.  
+To make DORA even more actionable, there are well-established benchmarks to determine if you are performing at "Elite", "High", "Medium", or "Low" levels. Inside DevLake, you will find the benchmarking table available to assess and compare your own projects.
 
 ## Why is DORA important?
-DORA metrics help teams and projects measure and improve software development practices to consistently deliver reliable products, and thus happy users!
 
+DORA metrics help teams and projects measure and improve software development practices to consistently deliver reliable products, and thus happy users!
 
 ## How to implement DORA metrics with Apache DevLake?
 
 You can set up DORA metrics in DevLake in a few steps:
+
 - **Install**: [Getting Started](https://devlake.apache.org/docs/GettingStarted)
 - **Collect**: Collect data via blueprint
-    - In the blueprint, select the data you wish to collect, and make sure you have selected the data required for DORA metrics
-    - Configure DORA-related transformation rules to define `deployments` and `incidents`
-    - Select a sync frequency for your data, save and run the blueprint.
+  - In the blueprint, select the data you wish to collect, and make sure you have selected the data required for DORA metrics
+  - Configure DORA-related transformation rules to define `deployments` and `incidents`
+  - Select a sync frequency for your data, save and run the blueprint.
 - **Report**: DevLake provides a built-in DORA dashboard. See an example screenshot below or check out our [live demo](https://grafana-lake.demo.devlake.io/grafana/d/qNo8_0M4z/dora?orgId=1).
-![DORA Dashboard](https://i.imgur.com/y1pUIsk.png)
+  ![DORA Dashboard](https://i.imgur.com/y1pUIsk.png)
 
 DevLake now supports Jenkins, GitHub Action and GitLabCI as data sources for `deployments` data; Jira, GitHub issues, and TAPD as the sources for `incidents` data; Github PRs, GitLab MRs as the sources for `changes` data.
 
-If your CI/CD tools are not listed on the [Supported Data Sources](https://devlake.apache.org/docs/SupportedDataSources) page, have no fear! DevLake provides incoming webhooks to push your `deployments` data to DevLake. The webhook configuration doc can be found [here](https://devlake.apache.org/docs/Configuration/webhook/).
-
+If your CI/CD tools are not listed on the [Supported Data Sources](https://devlake.apache.org/docs/next/Overview/SupportedDataSources) page, have no fear! DevLake provides incoming webhooks to push your `deployments` data to DevLake. The webhook configuration doc can be found [here](https://devlake.apache.org/docs/Configuration/webhook/).
```

Comment:
You can't use absolute URL of the next version of docs. You should remove the `/next/` part.

The solution is to use relative URL, then Docusaurus won't throw error and the URL will not break after the doc freeze.